tradeexecutor/visual/strategy_state.py

```diff
@@ -77,4 +166,4 @@ def draw_single_pair_strategy_state(
         volume_bar_mode=VolumeBarMode.hidden,  # TODO: Might be needed in the future strats
     )
 
-    return figure
+    return figure
```

Comment:
You can create a synthetic data strategy for testing. A strategy that jus buys/sells different pairs on different dates. See test_visualisations for examples.